sensei-media-attachments.php

```diff
@@ -1,13 +1,13 @@
 <?php
 /**
  * Plugin Name: Sensei LMS Media Attachments
- * Version: 2.0.2
+ * Version: 2.0.3
  * Plugin URI: https://woocommerce.com/products/sensei-media-attachments/
  * Description: Provide your students with easy access to additional learning materials, from audio files to slideshows and PDFs.
  * Author: Automattic
  * Author URI: https://automattic.com/
- * Requires at least: 5.0
- * Tested up to: 5.4
+ * Requires at least: 5.3
+ * Tested up to: 5.5
  * Requires PHP: 5.6
```

Comment:
We should probably bump this to 7.0 to match Sensei.

Reply:
I thought about that, but I didn't do it because the users still could update using it in old Sensei versions because we didn't add unsupported code here yet. But you're right, probably it's better to update it now: https://github.com/woocommerce/sensei-media-attachments/pull/98/commits/84f92c9a7d81553996a64b7884c7a16f08638121